apps/quarkus-full-microprofile/threshold.properties

```diff
@@ -1,6 +1,6 @@
 linux.time.to.first.ok.request.threshold.ms=300
 linux.RSS.threshold.kB=120000
-linux.executable.size.threshold.kB=73500
+linux.executable.size.threshold.kB=79000
```

Comment:
We need to make these Quarkus/Mandrel specific.

My understanding is that you are increasing the threshold for Quarkus 3.6, but this will make us miss increases happening in 3.2 for instance.

Reply:
+1

Reply:
@zakkak Yes. The increase is caused by OpenTelemetry being a much bigger library than OpenTracing apparently.

The "design" of the property files did not take into account the Q versions.

We can start having some per-quarkus-version property files. Perhaps per Mandrel major.minor too. 

Non-quarkus apps would have just the Mandrel major.minor?

Wondering how to make it nice and tidy without inflating the logic too much.

Reply:
> Non-quarkus apps would have just the Mandrel major.minor?

Yes

> Wondering how to make it nice and tidy without inflating the logic too much.

I need something similar in https://github.com/quarkusio/quarkus/pull/36108 and I was thinking about using a directory structure like `<quarkus-version>/<mandrel-version>/threshold.properties`. WDYT?